ovs/lib/storagerouter.py

```diff
@@ -1861,6 +1861,11 @@ def configure_disk(storagerouter_guid, disk_guid, partition_guid, offset, size,
                 if partition.mountpoint != mountpoint:
                     raise RuntimeError('Unexpected mountpoint')
             StorageRouterController._logger.debug('Mountpoint configured')
+        elif len(roles) == 0:  # Unmount disk without any roles assigned to be claimed by a backend
+            with remote(storagerouter.ip, [DiskTools], username='root') as rem:
+                StorageRouterController._logger.debug('Unmounting mountpoint: {0}'.format(partition.mountpoint))
+                rem.DiskTools.umount(mountpoint=partition.mountpoint)
```

Comment:
I'd rather not unmount the disk, we're not sure the customer doesn't use it for something else. Let's create a separate ticket where a customer can optionally check a box in the GUI where he indicates we can/should wipe the disk and - in case of role removal - unmount it.